internal/curves/curve.go

```diff
@@ -106,6 +106,16 @@ func (c functionSpeedCurve) Evaluate() (value int, err error) {
 	}
 
 	switch c.function {
+    case configuration.FunctionDelta:
+		var dmax = float64(values[0])
+		var dmin = float64(values[0])
+		for _, v := range values {
+			var val = float64(v);
```

Comment:
The semicolon is unnecessary,

and please avoid this temporary variable and just use `float64(v)` down below.